Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -57,10 +59,13 @@ + (NSString *)testEnvironmentWithConfiguration:(BPConfiguration *)config {
             [BPUtils printInfo:ERROR withString:[NSString stringWithFormat:@"Failed to load testcases from %@", [error localizedDescription]]];
             [BPUtils printInfo:WARNING withString:@"Will Run all TestCases"];
```

Comment:
This doesn't seem to be true anymore... if we can't load the xctest file we'll run the tests not skipped by testsCasesToSkip.